README

xiaoly10 opened the pull request:
zzz

hectorsector committed:
change title on README

newmanszhang commented:
zzz

newmanszhang commented:
```suggestion
You can play the game at: https://newmanszhang.github.io/github-games/
```

xiaoly10 reviewed (commented):
```suggestion
You can play the game at: https://newmanszhang.github.io/github-games/
```

xiaoly10 committed:
dd

```suggestion
You can play the game at: https://newmanszhang.github.io/github-games/
```
